src/Components/Web/src/Forms/InputLargeTextArea/InputLargeTextArea.cs

```diff
@@ -0,0 +1,149 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Text;
+using Microsoft.AspNetCore.Components.Rendering;
+using Microsoft.JSInterop;
+
+namespace Microsoft.AspNetCore.Components.Forms
+{
+    /// <summary>
+    /// A multiline input component for editing large <see cref="string"/> values, supports async
+    /// content access without binding nor validations.
```

Comment:
According to https://english.stackexchange.com/a/543921

```suggestion
    /// content access without binding and without validations.
```